### DISCUSSION TITLE
Manual/class-OcclusionPortal

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#682] [Notes]
# Manual/class-OcclusionPortal

https://docs.unity3d.com/Manual/class-OcclusionPortal.html

**Comment by mnicolas94** [comment]
The Center and Size fields of this class can't be accessed by code. Also, for some reason, they can't be accessed via reflection either. However, if you want to modify these values by code (editor only, tested on 2022.3.28) you can do so with the following code:
```
using (SerializedObject serializedObject = new SerializedObject(portal))
{
    serializedObject.Update();
    SerializedProperty centerProperty = serializedObject.FindProperty("m_Center");
    SerializedProperty sizeProperty = serializedObject.FindProperty("m_Size");
    centerProperty.vector3Value = <your-center-value>;
    sizeProperty.vector3Value = <your-size-value>;
    serializedObject.ApplyModifiedProperties();
}
```
